internal/db/group.go

```diff
@@ -96,6 +97,10 @@ func (d *Database) ForEachGroup(ctx context.Context, limit, offset int, f func(*
 	defer servermon.ErrorCounter(servermon.DBQueryErrorCount, &err, string(op))
 
 	db := d.DB.WithContext(ctx)
+	if match != "" {
+		match = "%" + match + "%"
+		db = db.Where("uuid LIKE ? OR name LIKE ?", match, match)
```

Comment:
Why not:
```go
db = db.Where("uuid = ? OR name = ?", match, match)
```
Because in theory I could just use % and get everything no?

I see Hue said fuzzy but I don't understand why we'd fuzzy match it. Also, names are unique so there's no reason to iterate over the group table when we can select where name = ?

Reply:
as discussed in a call, it's fine as it is. 
We need to check if they need fuzzy find on uuid as well